File: backend/src/main/java/com/NBE3_4_2_Team4/domain/payment/paymentCancel/repository/PaymentCancelRepository.java
package com.NBE3_4_2_Team4.domain.payment.paymentCancel.repository;

import com.NBE3_4_2_Team4.domain.payment.paymentCancel.entity.PaymentCancel;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PaymentCancelRepository extends JpaRepository<PaymentCancel, Long> {
}

File: backend/src/main/java/com/NBE3_4_2_Team4/domain/payment/paymentCancel/repository/PaymentCancelRepository.kt
package com.NBE3_4_2_Team4.domain.payment.paymentCancel.repository

import com.NBE3_4_2_Team4.domain.payment.paymentCancel.entity.PaymentCancel
import org.springframework.data.jpa.repository.JpaRepository

interface PaymentCancelRepository : JpaRepository<PaymentCancel, Long> {
}